### PR TITLE
2 cleanup commits

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: json_serializable/lib/src/type_helpers/to_from_string.dart [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+const uriString = ToFromString(
+  'Uri.parse',
+  'toString()',
+  TypeChecker.fromUrl('dart:core#Uri'),
+);
+
+/// Helper that unifies implementations of [Type]s that convert trivially
+/// to-from [String].
+class ToFromString {
+  final String _parse;
```
Comment (jakemac53) [inline json_serializable/lib/src/type_helpers/to_from_string.dart]: I would add some more docs here, these are string representations of the code to convert to/from strings yes?

File: json_serializable/lib/src/type_helpers/to_from_string.dart [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+const uriString = ToFromString(
+  'Uri.parse',
+  'toString()',
+  TypeChecker.fromUrl('dart:core#Uri'),
+);
+
+/// Helper that unifies implementations of [Type]s that convert trivially
+/// to-from [String].
+class ToFromString {
```
Comment (jakemac53) [inline json_serializable/lib/src/type_helpers/to_from_string.dart]: This name was really confusing to me at first, although I can't come up with a better one off the top of my head... maybe `StringSerializer` but that isn't really accurate either.

Comment (kevmoo) [reply json_serializable/lib/src/type_helpers/to_from_string.dart]: Made it `ToFromStringHelper` 🤷‍♂ – this is very package-internal

File: json_serializable/lib/src/type_helpers/to_from_string.dart (hunk 1 above)
Comment (kevmoo) [reply json_serializable/lib/src/type_helpers/to_from_string.dart]: done.

**Pull request merged**